Makefile

```diff
@@ -17,6 +17,15 @@ ISO ?= $(BUILD_ROOT)/metalk8s.iso
 CALICO_CNI_PLUGIN_VERSION = 3.5.1
 CALICO_CNI_PLUGIN_BUILD = 1
 
+CONTAINER_IMAGES = \
+	k8s.gcr.io/kube-apiserver@sha256:79b197b6a7334d2ad01ee82a9be82a39bfb50e33bc03263c58c4dfc4e81f71bc \
+	k8s.gcr.io/kube-controller-manager@sha256:bbb51bf83c73dfe8a924fd5f9e58f92c3feec44b24936f10b0087e8ffcd55f69 \
+	k8s.gcr.io/kube-scheduler@sha256:7a2a2a54f26647a4d8642d9158122e6c80e19b2b79d5bf0c74445f2e26a83dfa \
+	k8s.gcr.io/kube-proxy@sha256:8886e69a7946717a88cad9882e9b44927798d154088be19b5116a375c9923036 \
+	k8s.gcr.io/etcd@sha256:b960569ade5f37205a033dcdc3191fe99dc95b15c6795a6282859070ec2c6124 \
+	k8s.gcr.io/coredns@sha256:3e2be1cec87aca0b74b7668bbe8c02964a95a402e45ceb51b2252629d608d03a \
+	nginx@sha256:dd2d0ac3fff2f007d99e033b64854be0941e19a2ad51f174d9240dda20d9f534
```

Comment:
Not sure about the image of Nginx we should pull, this is the nginx:latest, maybe we want another one.

Reply:
The Makefile start to be a bit huge.

I would prefer to split the files based on their purpose.
a "container_images.mk" with the list.
Ideally, the file name are the same as the images (if that makes sense), so you can really trigger the right download for the given image if we change it

Reply:
done